test/e2e/main_test.go

```diff
@@ -766,7 +766,7 @@ func getSchemaForRest(ispn *ispnv1.Infinispan) string {
 		panic(err.Error())
 	}
 	kubernetes.Kubernetes.Client.Get(context.TODO(), types.NamespacedName{Namespace: ispn.Namespace, Name: ispn.Name}, &curr)
-	if curr.Spec.Security.EndpointEncryption.Type != "" {
+	if curr.Spec.Security.EndpointEncryption != nil && curr.Spec.Security.EndpointEncryption.Type != "" {
```

Comment:
We can introduce method like IsEndpointEncryptionTypeDefined

Reply:
ok, looks nice